server/backend/ios_notifications/ios_request_response/iosRequestResponseService.go

```diff
@@ -43,8 +51,6 @@ func (service *Service) HandleDeviceRequestResponse(request *pb.IOSDeviceRequest
 		return nil, ErrOutdatedRequest
 	}
 
-	influx.LogIOSBackgroundRequestResponse(requestLog.DeviceID, requestLog.RequestType)
```

Comment:
@Antonwy What do you need as a metric here exactly? (could you share the dashboards you build?)